network/src/peer_manager/senders.rs

```diff
@@ -47,6 +48,10 @@ impl PeerManagerRequestSender {
         protocol_id: ProtocolId,
         mdata: Bytes,
     ) -> Result<(), PeerManagerError> {
+        fail_point!("network::send::any", |_| {
```

Comment:
Is your goal to add this only for consensus? Or for all components? Right now, this will affect everything, right?
Also, why did you add this to the `PeerManagerRequestSender` and not at the network interface level? e.g., https://github.com/aptos-labs/aptos-core/blob/f005fa18b1095aad2ba4f128e5da88c89e4d164b/network/src/application/interface.rs#L166. 

If you want to add it to everything, the interface feels like a cleaner place?

Reply:
+1. But it looks like there is no easy way to add fail points only for consensus, without changing the existing networking?

Reply:
we can decide based on protocol id but probably not ideal

Reply:
I wanted to have an unreliable network - for everything, not just for consensus, so it is a realistic scenario. If RPC being delayed, blocks other RPCs from other protocols, and causes issues, we do want that.

for consensus only, we have consensus::send::any already


For why I placed it there, I wanted to have a separate decision for each destination peer, which is simpler to do in PeerManagerRequestSender, because network interface level does send_to_many in a single call

Reply:
This looks reasonable to me but wondering if we should use this for realistic workload scenario or use chaos monkey's network packet drop support. Later seems more realistic to me.